### PR TITLE
Pia 1889

### PR DESCRIPTION
**Pull request by zladzeyka**
GiniBankAPILibraryExample:
 - Add extraction feedback integration test.
 - Add test resources.

### CODE REVIEW

File: BankAPILibrary/GiniBankAPILibraryExample/GiniBankAPILibraryExampleTests/ExtractionFeedbackIntegrationTest.swift [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    func loadFile(withName name: String, ofType type: String) -> Data {
+        let fileURLPath: String? = Bundle.main
+            .path(forResource: name, ofType: type)
+        let data = try? Data.init(contentsOf: URL(fileURLWithPath: fileURLPath!))
+        
+        return data!
+    }
+    
+    func testSendExtractionFeedback() {
+
```
Comment (a-szotyori) [inline BankAPILibrary/GiniBankAPILibraryExample/GiniBankAPILibraryExampleTests/ExtractionFeedbackIntegrationTest.swift:36]: The other function bodies don't start with empty line. I'm fine both with or without the empty line. Just keep it consistent. 🤓

File: BankAPILibrary/GiniBankAPILibraryExample/GiniBankAPILibraryExampleTests/ExtractionFeedbackIntegrationTest.swift [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        let expect = expectation(description: "it logs the error event")
```
Comment (a-szotyori) [inline BankAPILibrary/GiniBankAPILibraryExample/GiniBankAPILibraryExampleTests/ExtractionFeedbackIntegrationTest.swift:37]: I am confused by the expectation description. Shouldn't it be something like: "feedback was correctly sent and extractions were updated"?

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Great job. I only have minor complaints. 🤓

File: BankAPILibrary/GiniBankAPILibraryExample/GiniBankAPILibraryExampleTests/ExtractionFeedbackIntegrationTest.swift (hunk 2 above)
Comment (zladzeyka) [reply BankAPILibrary/GiniBankAPILibraryExample/GiniBankAPILibraryExampleTests/ExtractionFeedbackIntegrationTest.swift]: 🙈 true

File: BankAPILibrary/GiniBankAPILibraryExample/GiniBankAPILibraryExampleTests/ExtractionFeedbackIntegrationTest.swift (hunk 1 above)
Comment (zladzeyka) [reply BankAPILibrary/GiniBankAPILibraryExample/GiniBankAPILibraryExampleTests/ExtractionFeedbackIntegrationTest.swift]: 👌🏻I'll fix it

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**